src/main/java/org/myrobotlab/service/Vertx.java

```diff
@@ -184,7 +196,34 @@ public boolean isLocal(Message msg) {
     return Runtime.getInstance().isLocal(msg);
   }
 
-  public io.vertx.core.Vertx getVertx() {
-    return vertx;
+  /**
+   * Restart service on different listening port
+   * 
+   * @param port
+   */
+  public void setPort(int port) {
+    config.port = port;
+    boolean wasListening = isListening;
+    if (isListening) {
+      stop();
+    }
+    sleep(2000);

```

Comment:
do we really need this sleep here?  shouldn't stop block until it's actually stopped?